pinot-segment-local/src/main/java/org/apache/pinot/segment/local/recordtransformer/TimeValidationTransformer.java

```diff
@@ -103,6 +103,7 @@ public GenericRow transform(GenericRow record) {
       if (_continueOnError) {
         LOGGER.debug(errorMessage);
         record.putValue(_timeColumnName, null);
+        record.putValue(GenericRow.INCOMPLETE_RECORD_KEY, true);
```

Comment:
Im fine with this change, but its better to check with the authors on why they did it this way. 

So with this change, we will skip the record (in NullValueTransformer), instead of assigning currentTime right?

Reply:
is this actually causing the even to be skipped? I tried tracing this, and I can't find anywhere this causes skipping

Reply:
This won't cause the row being skipped, but only log warnings on incomplete records. Time column will be filled with default value because the ingested value is invalid

Reply:
Updated the description. We do emit metric for real-time consumed rows